addons/point_of_sale/static/src/js/models.js

```diff
@@ -709,17 +701,28 @@ export class PosGlobalState extends PosModel {
     async _syncAllOrdersFromServer() {
         await this._removeOrdersFromServer();
         const ordersJson = await this._getOrdersJson();
-        await this._addPricelists(ordersJson);
-        await this._addFiscalPositions(ordersJson);
+        let message = false;
```

Comment:
Just a tip, prefer null, undefined or "". When seeing `message = false`, it implies that `message` should be a boolean.
Same for messageFp